### PR TITLE
Configuration/GitHub actions

### PR DESCRIPTION
**Pull request by LeandroSoria2**
### What changes does this PR include?
**Description**
config GitHub actions

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ module Target
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 7.0
+    config.autoloader = :classic
```
Comment (JulianPasquale) [inline config/application.rb]: We shouldn't do this. Rails 6 comes by default with Zeitwerk, and we should use that Autoloader

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+3.1.1
```
Comment (AgostinaDev) [inline .ruby-version]: Missing line at EOF 👀

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        uses: rootstrap/check_untracked_changes@v1
+        with:
+          path: "./db/schema.rb"
+      - name: Run Tests
+        run: |
+          bundle exec rspec       
+      - name: Check for missing annotations
+        run: bundle exec annotate
+      - name: Check for untracked changes in app and spec directories
+        uses: ./.github/actions/check_untracked_changes
```
Comment (AgostinaDev) [inline .github/workflows/ci.yml]: Be careful here, you're using this action from a different source in line 33

Comment (LeandroSoria2) [reply .github/workflows/ci.yml]: this was fixed with commit

File: config/application.rb (hunk 1 above)
Comment (LeandroSoria2) [reply config/application.rb]: It was fix with commit

**Review by sebastiancaraballo — APPROVED** [review]

**Review by JulianPasquale — APPROVED** [review]

**Review by AgostinaDev — APPROVED** [review]
🚀

File: .ruby-version (hunk 2 above)
Comment (LeandroSoria2) [reply .ruby-version]: It was fixed

**Pull request merged**